smdebug/tensorflow/keras.py

```diff
@@ -479,6 +495,7 @@ def on_epoch_end(self, batch, logs=None):
         if self._is_not_supported():
             return
         self._save_metrics(batch=batch, logs=logs, force_save=True)
+        self._save_model_outputs(logs=logs)
```

Comment:
if save tensor post step has this logic, why do we need to do it here?

Reply:
We do not need it here, since we currently save values produced during the train step.

Reply:
Why not other stages (EVAL & predict) ?